### PR TITLE
created listAllPrime.java

### PR DESCRIPTION
**Pull request by gane5hvarma**
lists out the prime numbers which are less than the input number

### CODE REVIEW

File: 17 primos/listAllPrime.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ 11
+ 13
+ 17
+ 19
+ 23
+ 29
+ */
+class prime{
+  int i;
+  prime(int a){
```
Comment (xaca) [inline 17 primos/listAllPrime.java]: Good, thank you!  What is the reason to put the code in the constructor? What do you think about make a static method in the class listAllPrime? Also remember java standar, the class name, must start with a capital letter.

Comment (gane5hvarma) [reply 17 primos/listAllPrime.java]: My bad i started learning java this semester and yeah i put the code in the constructor because it looks good.

**Comment by gane5hvarma** [comment]
thanks man got to learn something new using static

File: 17 primos/listAllPrime.java (hunk 1 above)
Comment (xaca) [reply 17 primos/listAllPrime.java]: Constructor is for initialize variables or call class methods. Your code works, but my question is about software design. I glad to help you to improve your programming skils, I recomend read the book clean code.

**Comment by xaca** [comment]
@gane5hvarma Great, We like to write good code. We need to follow some rules, java have a lot.

File: 17 primos/listAllPrime.java [hunk 2]
```diff
@@ -28,12 +28,12 @@
...
   }
-
-}
-class ListAllPrime{
+ 
   public static void main(String args[]){
     int m;
     m=Integer.parseInt(args[0]);
-    Prime p=new Prime(m);
+    ListAllPrime.prime(m);
```
Comment (xaca) [inline 17 primos/listAllPrime.java]: prime is static, so you could call only using the method name (prime), because main is static to.

**Pull request merged**